dolittle/Studio
# Fixing icon to look the same as figma

## Summary

ERP integration icons match that in figma

Linked to https://app.asana.com/0/1201028542784316/1201126860304301/f

# How to test
- Fire up studio locally
- Click "Microservices"
- Click "Create Microservices"
- Click "Purchase Order"
- Observe icons look the same as below

![Uploading Screenshot 2021-10-05 at 16.56.23.png…]()
